chart/templates/workers/worker-deployment.yaml

```diff
@@ -214,6 +214,9 @@ spec:
           {{- include "custom_airflow_environment" . | indent 10 }}
           {{- include "standard_airflow_environment" . | indent 10 }}
         {{- end }}
+{{- if .Values.workers.extraContainers }}
+{{- toYaml .Values.workers.extraContainers | nindent 8 }}
```

Comment:
Should we also update `pod-template-file.kubernetes-helm-yaml` to support Kubernetes Workers?
https://github.com/apache/airflow/blob/master/chart/files/pod-template-file.kubernetes-helm-yaml

Reply:
Not tested with Kubernetes workers.
The `extraVolumes` and `extraVolumeMounts` options also seem to be excluded from the file.
Need to support it?

Reply:
For now, we can only limit ourselves to Celery Executor, but I think it is worth adding an entry in the documentation for this limitation.

Reply:
@Swalloow Can you add the limitation to the docs